.github/workflows/npmpublish.yml

```diff
@@ -1,38 +1,39 @@
-name: "NPM Publish and GitHub tag"
+name: 'NPM Publish and GitHub tag'
```

Comment:
This workflow must change because we now have two packages that are published to npm, `markdown-it-synapse` and `synapse-react-client`